### DISCUSSION TITLE
create tables in a new pdf?

### DISCUSSION THREAD

**Discussion by markat1** [UglyToad/PdfPig#933] [Q&A] [answered]
I couldn't find documentation mentioning wether it's possible to create tables for a new pdf that I'm building.

is it possible I can create tables with pdfpig? if yes, could you provide me an example, please.

**Comment by markat1** [comment]
hmm I guess maybe https://github.com/BobLd/tabula-sharp  can help me? eventhough it seems to primary be for exacting data from tables.

**Reply by BobLd** [reply, marked as answer]
hi @markat1, tabula-sharp is only for extracting tables, and not for table creations.

The concept of 'table' does not exist in pdf (hence the need to libraries to extract tabular data)... In order for you to display a table in a pdf document, you will need to draw the line and position the text yourself.

What could be added in PdfPig is a helper class top make it easier to draw table, but this is not a short term plan.

**Reply by markat1** [reply]
Thank you so much for your reply @BobLd ! any examples on how to draw a table with PdfPig?
